src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -1002,14 +931,15 @@ private void determineUnitsToAttackWith(final Map<Territory, ProTerritory> attac
     }
   }
 
-  private Map<Unit, Set<Territory>> tryToAttackTerritories(final Map<Territory, ProTerritory> attackMap,
-      final ProMoveOptions enemyAttackOptions, final Map<Unit, Set<Territory>> unitAttackMap,
-      final List<ProTerritory> prioritizedTerritories, final List<ProTransport> transportMapList,
-      final Map<Unit, Set<Territory>> transportAttackMap, final Map<Unit, Set<Territory>> bombardMap,
+  private Map<Unit, Set<Territory>> tryToAttackTerritories(final List<ProTerritory> prioritizedTerritories,
       final List<Unit> alreadyMovedUnits) {
 
-    final Map<Unit, Territory> unitTerritoryMap = ProUtils.createUnitTerritoryMap(player);
-    final IntegerMap<UnitType> playerCostMap = BattleCalculator.getCostsForTUV(player, data);
+    final Map<Territory, ProTerritory> attackMap = territoryManager.getAttackOptions().getTerritoryMap();
```

Comment:
Exactly. You've hit on that next phase. I've essentially consolidated all the maps/lists into one place but now need to limit access to them.